AzureCalling/Controllers/BottomDrawerViewController.swift

```diff
@@ -32,24 +35,32 @@ class BottomDrawerViewController: UIViewController {
         view.isOpaque = false
 
         // tapping anywhere on the view is the same as tapping cancel
-        let tap = UITapGestureRecognizer(target: self, action: #selector(dismissSelf))
+        let tap = UITapGestureRecognizer(target: self, action: #selector(gestureRecognizer))
         tap.cancelsTouchesInView = false
+        tap.delegate = self
         view.addGestureRecognizer(tap)
-        view.isUserInteractionEnabled = true
 
         createBottomDrawer()
     }
 
-    @objc func dismissSelf() {
-        dismiss(animated: true, completion: nil)
-    }
-
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
 
         openBottomDrawer()
     }
 
+    // MARK: UIGestureRecognizerDelegate events
+
+    @objc func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
```

Comment:
Is this necessary? If we don't want to dismiss the viewcontroller when user tap the tableviewcell, could we just set tableview.view.isUserInteractionEnabled to false?

Reply:
Tried `tableView.isUserInteractionEnabled = false`, still triggers `UITapGestureRecognizer` event when tapping the bottom drawer